samples/v1.0/Scenarios/InputForm.json

```diff
@@ -31,35 +31,25 @@
 							"wrap": true,
 							"size": "small"
 						},
-						{
-							"type": "TextBlock",
-							"text": "Your name",
-							"wrap": true
-						},
 						{
 							"type": "Input.Text",
 							"id": "myName",
+							"label": "Your name",
 							"placeholder": "Last, First"
 						},
-						{
-							"type": "TextBlock",
-							"text": "Your email",
-							"wrap": true
-						},
 						{
 							"type": "Input.Text",
 							"id": "myEmail",
+							"label": "Your email",
+							"regex": "^(.+)@(.+)$",
```

Comment:
I've added a very simple regex here for the email validation (it is just a sample after all), but I don't think there is an easy regex for phone number or full name, so those won't have error messages.

Reply:
**Regex for phone number**
- Try this: `/^\\(?([0-9]{3})\\)?[-.\\s]?([0-9]{3})[-.\\s]?([0-9]{4})$/` ([Source](https://regexpattern.com/phone-number/))
- I would rather we have something than nothing here (don't want them to send a similar bug back). It doesn't have to be perfect though. At the end of the day, this is a sample that the author can build off of and introduce their own regex if they want.

**Regex for name**
- I don't think this needs/deserves validation, so I'm fine with it.

Reply:
Yeah I personally wasn't a fan of putting a regex for the phone number that was region-specific but sure, its just a sample. Added it!